docs/src/installation.md

```diff
@@ -72,6 +72,19 @@ Such solvers include [Bonmin](https://github.com/coin-or/Bonmin) and
 [Couenne](https://github.com/coin-or/Couenne). See a more complete list
 [here](https://ampl.com/products/solvers/all-solvers-for-ampl/).
 
+Additionally, you may use [GAMS](https://github.com/GAMS-dev/gams.jl) to access additional solvers. Among them are 
+[AlphaECP](https://www.gams.com/latest/docs/S_ALPHAECP.html), 
+[Antigone](https://www.gams.com/latest/docs/S_ANTIGONE.html), 
+[BARON](https://www.gams.com/latest/docs/S_BARON.html), 
+[CONOPT](https://www.gams.com/latest/docs/S_CONOPT.html), 
+[Couenne](https://www.gams.com/latest/docs/S_COUENNE.html), 
+[LocalSolver](https://www.gams.com/latest/docs/S_LOCALSOLVER.html), 
+[PATHNLP](https://www.gams.com/latest/docs/S_PATHNLP.html),
+[SHOT](https://www.gams.com/latest/docs/S_SHOT.html), 
+[SNOPT](https://www.gams.com/latest/docs/S_SNOPT.html), 
+[SoPlex](https://www.gams.com/latest/docs/S_SOPLEX.html). 
+See a complete list [here](https://www.gams.com/latest/docs/S_MAIN.html).
```

Comment:
Could you add a note that GAMS is commercial software that requires a license? We make this distinction in the solver table as well.